fail2ban: include README files

iigs opened the pull request:
Include README files missing from initial package.

Fix comments in Makefile.

iigs committed:
fail2ban: include README files

mtelka commented on components/network/fail2ban/Makefile:
```diff
@@ -1,26 +1,33 @@
-# Common Development and Distribution License ("CDDL"), version 1.0.
-# You may only use this file in accordance with the terms of version
-# 1.0 of the CDDL.
+# CDDL HEADER START
```
Why this header change?  We prefer the previous one.  See https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/templates

mtelka commented on components/network/fail2ban/Makefile:
```diff
@@ -1,26 +1,33 @@
 #
-
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
```
... because this URL is not reachable, for example.

mtelka commented on components/network/fail2ban/Makefile:
```diff
@@ -63,7 +71,6 @@ COMPONENT_POST_INSTALL_ACTION += \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-freebsd.conf ; \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-opensuse.conf ; \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-osx.conf ; \
```
You want to remove the ending backslash too.

iigs commented:
What are the odds that I copy the CDDL header from a random Makefile in the repo and it is the older style one?  Copied header from a template .mk file. Removed backslash.  Changed the foo=bar lines to match a template .mk file.
